client/src/assets/components/subcomponents/Register.jsx

```diff
@@ -111,8 +114,65 @@ const Register = () => {
     formData.matchingPassword,
   ]);
 
+  useEffect(() => {
+    if (!usernameFocus || !username) return;
+    console.log("formData.username", username, formData.username);
```

Comment:
I understand this for testing reason but can be removed once ready to implement properly.